src/traffic_ctl/CtrlCommands.h

```diff
@@ -148,18 +166,26 @@ class HostCommand : public CtrlCommand
 class PluginCommand : public CtrlCommand
 {
 public:
-  PluginCommand(ts::Arguments args);
+  PluginCommand(ts::Arguments *args);
 
 private:
+  static inline const std::string MSG_STR{"msg"};
   void plugin_msg();
 };
 // -----------------------------------------------------------------------------------------------------------------------------------
 class DirectRPCCommand : public CtrlCommand
 {
 public:
-  DirectRPCCommand(ts::Arguments args);
+  DirectRPCCommand(ts::Arguments *args);
 
 private:
+  static inline const std::string GET_API_STR{"get-api"};
```

Comment:
You should make these `string_view` and avoid the allocation and copy, even if it's just once per process.